content/docs/services/cloud-gov-service-account.md

```diff
@@ -60,9 +88,6 @@ If you're trying to retrieve credentials for a service instance created before J
 
 To use this service, see [continuous deployment]({{< relref "docs/apps/continuous-deployment.md" >}}).
 
-### Rotating credentials
```

Comment:
We include this section for most service pages, so it might more sense to keep the usual section heading and move extra information here. WDYT @brittag ?

Reply:
We use "Rotating" in our operations docs, but I think "updating" is more applicable to our audience. I don't see any example of the term "rotating" in customer-facing docs; LMK if I'm mistaken.

Reply:
Sure, let's use rotate for consistency.

Reply:
Consistency in headers is great. Thanks!